_posts/2019-08-26-values.md

```diff
@@ -61,6 +61,21 @@ layout: null
     </div>
 </div>
 
+<h1><a id="#value_ifelse">If - Then - Else</a></h1>
+<div id="wrapper">
+    <div id="first"> 
+    <img src="{{'/guide/images/value_ifelse.png'| relative_url}}" width="100%">
+    </div>
+    <div id="second">
+    <h3>Description: The If - Then - Else block first checks its boolean condition. If the condition is true, the code held inside the Then space will activate, and the script will continue. If the condition is false, the code inside the Else space will activate.</h3>
```

Comment:
Should probably say that this is done row-by-row - maybe a small HTML table showing an example?